### DISCUSSION TITLE
Which Structured requests will give me few ietms instead of one ?

### DISCUSSION THREAD

**Discussion by BobSquarePants** [osm-search/Nominatim#3035] [Q&A] [answered]
Hi everyone,

I'm using the [Structured requests](https://nominatim.org/release-docs/develop/api/Search/#parameters) and in my early tests, I have always a return of one items (dict)

With a `Structured requests` is possible to get more that one item ? if yes, do you have an example ? I would like to test it.

Thanks.

**Comment by mtmail** [comment, marked as answer]
You can test structured search on https://nominatim.openstreetmap.org/ui/search.html I don't believe it leads to more search results.

**Reply by BobSquarePants** [reply]
Thank you @mtmail !
I didn't know that url, (click, bookmarked.. :) )
